### ISSUE TITLE
Add Windows ARM64 as a target platform for tests and build

### ISSUE THREAD

**Issue by remyroy** [ethstaker/ethstaker-deposit-cli#160]
Related to #26 

Part of the work was started by @yorickdowne in #132 . I believe we depends on Github infra team or someone related to them to create a proper Windows ARM64 image with build tools preinstalled.

**Comment by yorickdowne** [comment]
Correct, we are waiting for a new Windows arm64 runner image. It should be available Soon(tm) :harold

Once we have that, we can test CI again.
